src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -403,31 +403,78 @@ private static string GetParameterText(string parameterName)
 
                 CommandProcessorBase commandProcessor = null;
                 CommandRedirection[] commandRedirection = null;
-                for (int i = 0; i < pipeElements.Length; i++)
+
+                if (((PipelineAst)pipeElementAsts[0].Parent).BackgroundProcess)
                 {
-                    commandRedirection = commandRedirections != null ? commandRedirections[i] : null;
-                    commandProcessor = AddCommand(pipelineProcessor, pipeElements[i], pipeElementAsts[i],
-                                                  commandRedirection, context);
+                    // For background jobs rewrite the pipeline as a Start-Job command
+                    var scriptblockBodyString = ((PipelineAst)pipeElementAsts[0].Parent).Extent.Text;
+                    Token[] tokens;
+                    ParseError[] errors;
+                    var pipelineBodyAst = Parser.ParseInput(scriptblockBodyString, out tokens, out errors);
+                    // Prefix variables in the scriptblock with $using: 
+                    var variables = pipelineBodyAst.FindAll(x => x is VariableExpressionAst, true).Reverse();
+                    foreach (var v in variables)
+                    {
+                        var vName = v.Extent.Text.Substring(1);
+                        // Skip variables that don't exist
```

Comment:
You generally don't want to use Extent.Text except for error messages or other ui purposes, it might include more than you think, e.g. `${pid}` - it includes the braces.

Instead, you should use the VariablePath property.